uPortal-portlets/src/main/java/org/apereo/portal/portlets/portletadmin/PortletAdministrationHelper.java

```diff
@@ -268,7 +279,9 @@ private PortletDefinitionForm createNewPortletDefinitionForm() {
      */
     public PortletDefinitionForm savePortletRegistration(
             IPerson publisher, PortletDefinitionForm form) throws Exception {
-
+        if (this.logger.isTraceEnabled()) {
+            logger.trace("In savePortletRegistration() - for: " + form.getPortletName());
+        }
```

Comment:
Can you take a moment to update the class to `org.slf4j.Logger`, which doesn't need any of that `isTraceEnabled()` nonsense?